docs/install.rst

```diff
@@ -125,12 +125,12 @@ Method 2: Install from Source
 
 Check Installation
 ------------------
-You can check the installation by running the following commands.
+You can check the installation by running the following commands from the root of the repository.
 
 .. code:: bash
 
   ray start --head
-  python3 -m alpa.test_install
+  python3 tests/test_install.py
```

Comment:
`python3 tests/test_install.py` is deprecated.
`python3 -m alpa.test_install` is recommended because it also works if people instal by wheel